robotiq_c_model_control/nodes/test_node.py

```diff
@@ -0,0 +1,175 @@
+"""
+Copyright 2018 by READY Robotics Corporation.
+All rights reserved. No person may copy, distribute, publicly display, create derivative works from or otherwise
+use or modify this software without first obtaining a license from the READY Robotics Corporation.
+"""
+import rospy
+from actionlib.simple_action_client import SimpleActionClient
+from robotiq_c_model_control.msg import (
+    GripperAction,
+    GripperGoal
+)
+from ready_debug_tools import ipdb_helpers
+
+def main():
+    rospy.init_node('dual_gripper_test_node')
+
+    grip_a_ac = SimpleActionClient('/robotiq_grip_a/grip_action', GripperAction)
```

Comment:
Longer term: Use enumeration idiom used elsewhere in the system. 
```suggestion
    grip_a_ac = SimpleActionClient('/robotiq_grip/A/grip_action', GripperAction)
```

Reply:
Deferred until develop merge. See here: https://readyrobotics.atlassian.net/browse/RPI-33